tests/Integration/config/packages/5.3/framework.yaml

```diff
@@ -0,0 +1,17 @@
+framework:
+    secret: '%env(APP_SECRET)%'
+    #csrf_protection: true
```

Comment:
Is it necessary to keep this commented line?

Reply:
Not at all, came from the recipe I copied from Symfony's repository.

I will clean this